moveit_core/robot_model/include/moveit/robot_model/robot_model.h

```diff
@@ -428,7 +428,7 @@ class RobotModel
     return variable_names_;
   }
 
-  /** \brief Get the bounds for a specific variable. Throw an exception of variable is not found. */
```

Comment:
I happened to dig through the code and did not see a place where an exception is thrown

Reply:
If I were reading the docs I'd be curious to know what does happen if the variable is not found. It can't return a nullptr due to the ref return type, so does it just return garbage or worse segfault?

Reply:
OK, I was wrong here. It does throw but you have to dig several layers deep to find it. Will revert this change.

https://github.com/ros-planning/moveit2/blob/002dcbf82d0b95a28ac17048c16b8f00679d0e66/moveit_core/robot_model/src/joint_model.cpp#L89